article/app/model/LiveBlogHelpers.scala

```diff
@@ -51,6 +61,8 @@ object LiveBlogHelpers {
       liveBlog: Article,
       response: ItemResponse,
       range: BlockRange,
+      shouldFilter: Boolean,
```

Comment:
```suggestion
      filterSwitch: Boolean,
```

Reply:
And everywhere relevant

Reply:
@abeddow91

Reply:
resolved in https://github.com/guardian/frontend/pull/24341